### DISCUSSION TITLE
Spreadsheet and Worksheet class are disappared in api document

### DISCUSSION THREAD

**Discussion by non1979cn** [PHPOffice/PhpSpreadsheet#2140] [Q&A] [answered]
in the API documentation - https://phpoffice.github.io/PhpSpreadsheet/, I have see the spreedsheet and worksheet class reference， but now， they have missed，how can I find them?

**Comment by MarkBaker** [comment, marked as answer]
We have updated the version of PHPDocumentor that generates this files; hopefully that will resolve the problem

**Reply by non1979cn** [reply]
yes,I have see it, thank a lot
